src/test/scala/com/microsoft/ml/spark/nbtest/SynapseUtilities.scala

```diff
@@ -0,0 +1,286 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark.nbtest
+
+import com.microsoft.ml.spark.Secrets
+import com.microsoft.ml.spark.build.BuildInfo
+import com.microsoft.ml.spark.cognitive.RESTHelpers
+import com.microsoft.ml.spark.core.env.FileUtilities
+import org.apache.commons.io.IOUtils
+import org.apache.http.client.entity.UrlEncodedFormEntity
+import org.apache.http.client.methods.{HttpDelete, HttpGet, HttpPost}
+import org.apache.http.entity.StringEntity
+import org.apache.http.message.BasicNameValuePair
+import org.json4s.JsonAST.JObject
+import org.json4s.jackson.JsonMethods.parse
+import org.json4s.jackson.Serialization
+import org.json4s.jackson.Serialization.write
+import org.json4s.{Formats, NoTypeHints}
+import spray.json.DefaultJsonProtocol._
+import spray.json._
+
+import java.io.{File, InputStream}
+import java.util
+import scala.annotation.tailrec
+import scala.concurrent.{TimeoutException, blocking}
+import scala.io.Source
+import scala.language.postfixOps
+import scala.sys.process._
+
+
+case class LivyBatch(id: Int,
+                     state: String,
+                     appId: Option[String],
+                     appInfo: Option[JObject],
+                     log: Seq[String])
+
+case class LivyBatchJob(livyBatch: LivyBatch,
+                        sparkPool: String,
+                        livyUrl: String)
+
+case class Application(state: String,
+                       name: String,
+                       livyId: String)
+
+case class Applications(nJobs: Int,
+                        sparkJobs: Seq[Application])
+
+//noinspection ScalaStyle
```

Comment:
what style check are we trying to avoid here?

Reply:
removed this line.